### PR TITLE
changes required for a new hub deployment

### PR DESCRIPTION
**Pull request by sooyounga**
from trying to deploy a new hub separately for the first time, discovered these changes to be made: 
- jupyterhub chart version update
- helm upgrade 
- kubernetes library import renamed to kubernetes_asyncio (from jupyterhub version update)
- singleuser_image_spec renamed to image (from new kubespawner version, @satra can you confirm you made this diff from dandi hub to kb hub? i double-checked from rerunning it but just to confirm)
- debug mode enabled 
- increased timeout duration

https://docs.google.com/document/d/1SC08Vd38z3vGfaYGYZYwdCN_ZHRBWpSHkb4aQ9JzQ1o/edit#heading=h.i3nokrj0xsid

### CODE REVIEW

**Comment by sooyounga** [comment]
updated README to remove dandi-info subfolder texts

File: z2jh.yml [hunk 1]
```diff
@@ -469,7 +469,7 @@
         changed_when: False
 
       - name: Install JupyterHub release
-        shell: helm install {{ namespace }}-jupyterhub {{ helm_chart_repo_name }}/jupyterhub \
+        shell: helm upgrade --install {{ namespace }}-jupyterhub {{ helm_chart_repo_name }}/jupyterhub \
```
Comment (satra) [inline z2jh.yml:472]: is there a reason for this change. upgrade shouldn't be required the first time.

Comment (satra) [inline z2jh.yml:472]:
```suggestion
        shell: helm --install {{ namespace }}-jupyterhub {{ helm_chart_repo_name }}/jupyterhub \
```

File: z2jh.yml [hunk 2]
```diff
@@ -469,7 +469,7 @@
         changed_when: False
 
       - name: Install JupyterHub release
-        shell: helm install {{ namespace }}-jupyterhub {{ helm_chart_repo_name }}/jupyterhub \
+        shell: helm --install {{ namespace }}-jupyterhub {{ helm_chart_repo_name }}/jupyterhub \
```
Comment (satra) [inline z2jh.yml:472]:
```suggestion
        shell: helm install {{ namespace }}-jupyterhub {{ helm_chart_repo_name }}/jupyterhub \
```

File: config.yaml.j2 [hunk 3]
```diff
@@ -133,6 +133,9 @@ cull:
   timeout: 3600
   every: 300
 
+debug:
+  enabled: true
+
```
Comment (satra) [inline config.yaml.j2:138]:
```suggestion
```

**Pull request merged**